plugins/BEdita/API/src/Auth/EndpointAuthorize.php

```diff
@@ -178,12 +182,21 @@ protected function getPermissions($user, Application $application, Endpoint $end
      */
     protected function checkPermissions(array $permissions)
     {
+        $result = EndpointPermission::PERM_NO;
+        if (empty($permissions)) {
+            $count = TableRegistry::get('EndpointPermissions')->find()->count();
```

Comment:
Count should be done on current endpoint not on `any` endpoint
If I set a permission only on `/roles` endpoint I will get `401 Unauthorized` also trying to access `/home`

What I would do on empty `$permissions`:
  * if `$endpoint` is `null` (no endpoint and no permissions found) => YES
  * if `$endpoint` is set and no permissions are set for that endpoint (`$count === 0`) => YES
  * otherwise => NO